docs/UsersGuide/source/WE2Etests.rst

```diff
@@ -1,212 +1,179 @@
+
+The script to run the WE2E tests is named ``run_WE2E_tests.sh`` and is located in the directory ``ufs-srweather-app/regional_workflow/tests/WE2E``. Each WE2E test has an associated configuration file named ``config.${test_name}.sh``, where ``${test_name}`` is the name of the corresponding test. These configuration files are subsets of the full ``config.sh`` experiment configuration file used in :numref:`Section %s <SetUpConfigFileC>` and described in :numref:`Section %s <UserSpecificConfig>`. For each test, the ``run_WE2E_tests.sh`` script reads in its configuration file and generates from it a complete ``config.sh`` file. It then calls ``generate_FV3LAM_wflow.sh``, which in turn reads in ``config.sh`` and generates a new experiment for the test. The name of each experiment directory is set to that of the corresponding test, and a copy of ``config.sh`` for each test is placed in its experiment directory.
 
 Since ``run_WE2E_tests.sh`` calls ``generate_FV3LAM_wflow.sh`` for each test, the 
 Python modules required for experiment generation must be loaded before ``run_WE2E_tests.sh`` 
-can be called.  See :numref:`Section %s <SetUpPythonEnv>` for information on loading the Python
-environment on supported platforms.  Note also that ``run_WE2E_tests.sh`` assumes that all of 
-the executables have been built.  If they have not, then ``run_WE2E_tests.sh`` will still
+can be called. See :numref:`Section %s <SetUpPythonEnv>` for information on loading the Python
+environment on supported platforms. Note also that ``run_WE2E_tests.sh`` assumes that all of 
+the executables have been built. If they have not, then ``run_WE2E_tests.sh`` will still
 generate the experiment directories, but the workflows will fail.
 
+Supported Tests
+===================
+
+The full list of WE2E tests is extensive; it is not recommended to run all of the tests, as some are computationally expensive. A subset of the tests are supported for the latest release of the SRW Application`. Frequent test cases appear in :numref:`Table %s <FrequentTests>` below, and complete test cases appear :doc:`here <CompleteTests>`. Frequent tests are a lightweight set of tests that can be automated and run regularly on each Level 1 platform. These tests and cover a wide scope of capabilities to ensure that there are no major, obvious faults in the underlying code. Complete tests include the remainder of the supported WE2E tests and cover a fuller list of workflow configurations.
+
+..
+   COMMENT: Add file w/supported tests to repo 
+   COMMENT: Contrib Guide says that "Fundamental testing consists of a lightweight set of tests that can be automated and run regularly on each Level 1 platform. These are mostly low-resolution tests and cover a wide scope of capabilities to ensure that there are no major, obvious faults in the underlying code. Comprehensive testing includes the entire set of WE2E tests." How would we define frequent v complete? 
+
```

Comment:
"Frequent" tests would be nightly or weekly.  A "complete" set of tests would be the comprehensive, full suite of WE2E tests.  @mkavulich, thoughts here?